src/app/core/common-components/entity-select/entity-select.component.spec.ts

```diff
@@ -36,6 +38,8 @@ describe("EntitySelectComponent", () => {
         ]),
       ],
     }).compileComponents();
+    entityAbility = TestBed.inject(EntityAbility);
+    spyOn(entityAbility, "can").and.returnValue(true);
```

Comment:
Thats a bit annoying, maybe we should add the default "everything is allowed"-rule to the `MockedTestingModule`.

Reply:
Problem is also that our "unit tests" are more like integration tests with a lot of real dependencies... But yes, the default could be to allow everything.